### ISSUE TITLE
Anyway to confirm the bias-T of a RTL-SDR v3 is turned on or not? 

### ISSUE THREAD

**Issue by shuwang1** [rtlsdrblog/rtl_biast#4]
Hello,

The installation worked and not error information come out.  Nevertheless, how can I know if the biasT is really on?

Thx!

Shu

**Comment by mash55** [comment]
voltmeter/multimeter

**Comment by rtlsdrblog** [comment]
Yep voltmeter, or most LNAs have an LED which should indicate power.
